packages/dashboard/src/components/three-fiber/lift-three.tsx

```diff
@@ -1,3 +1,4 @@
+import robotoFont from '@fontsource/roboto/files/roboto-latin-400-normal.woff2';
```

Comment:
running `hotel` world, the map loads and crashes, I'm getting this error

![image](https://github.com/user-attachments/assets/290ae510-2ec2-41e0-bba0-8bb3ab966ee7)

this doesn't happen in `office` world since it doesn't have lifts

Reply:
I don't get the error in hotel world, might be a browser thing, changed to woff for compatibility.